minifi-nar-bundles/minifi-framework-bundle/minifi-framework/minifi-runtime/pom.xml

```diff
@@ -40,6 +40,24 @@ limitations under the License.
             <groupId>org.apache.nifi.minifi</groupId>
             <artifactId>minifi-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>ro.pippo</groupId>
+            <artifactId>pippo-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>ro.pippo</groupId>
+            <artifactId>pippo-tomcat</artifactId>
```

Comment:
Ah gotcha. In the interim as we are kind of prototyping things out let's not go through solving time sinks for "optimization" at this juncture. Maybe make another ticket to address at a later time? Implementation specifics are hidden behind the interface so should be okay to swap.